openc3-cmd-tlm-api/app/models/screen.rb

```diff
@@ -105,4 +105,12 @@ def self.create(scope, target, screen, text = nil)
       content_type: 'text/plain')
     true
   end
+
+  def self.destroy(scope, target, screen)
+    rubys3_client = Aws::S3::Client.new
+    rubys3_client.delete_object(
+      # Only delete from targets_modified
+      key: "#{scope}/targets_modified/#{target}/screens/#{screen}.txt",
+      bucket: DEFAULT_BUCKET_NAME)
+  end
```

Comment:
This is a little weird because if you delete an original plugin screen nothing happens